bundle/compliance/cis_eks/data_adapter.rego

```diff
@@ -3,15 +3,15 @@ package compliance.cis_eks.data_adapter
 import data.compliance.lib.data_adapter
 
 is_aws_eks {
-	input.type == "aws-eks"
+	input.type == "container-management"
```

Comment:
shouldn't the validations be on the more aws-specific subtype?

Reply:
👍

Reply:
Let's pass `sub_type` to OPA then.
Current names don't match what @tinnytintin10 was aiming for.